Resources/Prototypes/Roles/Jobs/Fun/misc_startinggear.yml

```diff
@@ -135,8 +135,6 @@
     pocket1: DoubleEmergencyOxygenTankFilled
     pocket2: BaseUplinkRadio40TC
     belt: ClothingBeltMilitaryWebbing
-  inhand:
```

Comment:
All you would need is to comment out those two lines and keep the rest.
This technically allows the least amount of conflicts and admins can still spawn them in if they want to for events.

Reply:
This point is still valid. Don't remove the lines, just comment them out.